### PR TITLE
Refactor/fixes db

### PR DESCRIPTION
**Pull request by bhavyaj12**
### Added sort by filter on user home feed page, other page posts sorted, added change profile image
- Added more post data
- Corrected profile pictures in comments, added profile pic in db for comments
- Removed mockman
- Some CSS fixes for profile picture

**Please check app is RESPONSIVE**

### What not to check
- Search

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
looks good to be merged

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks good code well structure

**Review by shubhamkumar648 — APPROVED** [review]
- Nice ui design 
- code look consisitent 
- good to merge

File: src/components/CommentForm.jsx [hunk 1]
```diff
@@ -152,7 +154,7 @@ const CommentForm = ({ post }) => {
               >
                 <div className="d-flex justify-content-start align-items-center">
                   <Image
-                    src="https://www.shareicon.net/data/128x128/2016/07/05/791214_man_512x512.png"
+                    src={commentItem.username === user.username ? user.profileImg : commentItem.profileImg}
```
Comment (shashikantmaurya0007) [inline src/components/CommentForm.jsx:157]: extract this logic to a different function

**Pull request merged**